src/Microsoft.ML/CSharpApi.cs

```diff
@@ -3642,15 +3642,18 @@ namespace Trainers
     {
 
         /// <summary>
-        /// Perceptron is a classification algorithm that makes its predictions based on a linear function. 
+        /// Perceptron is a classification algorithm that makes its predictions based on a linear function.
         /// I.e., for an instance with feature values f0, f1,..., f_D-1, , the prediction is given by the sign of sigma[0,D-1] ( w_i * f_i), where w_0, w_1,...,w_D-1 are the weights computed by the algorithm.
+        /// 
```

Comment:
:memo: blank lines are ignored in XML (they all collapse to a single space). If you want paragraphs, each block should start with `<para>` and end with `</para>`.